.env.dev

```diff
@@ -1,26 +1,28 @@
-FRONTEND_PORT="3000"
```

Comment:
The change in the formatting of the values (the "" ) is not necessary.

Usually, the only changes to a .env file should be adding / removing variables. It is best to leave the formatting as-is.